### ISSUE TITLE
[Improvements] Better Message Deletion

### ISSUE THREAD

**Issue by ghost** [GooseMod-Modules/Requests#4]
Original:
I know there is already a module for deleted messages but:
- The message goes away after you restart discord
- There could be a option so instead of the whole message going red only the text would be red
- It would be cool if it also saved edited messages so you have the message before being edited and then down in the same "block" of the message it would show the edited message, and the message before being edited would get like a gray color with a lower opacity, maybe you could make this part customizable.
- And a option to delete or clear all the saved messages (deleted and edited) 

Edited todo:
 - [ ] Add text going red option instead of background

**Comment by CanadaHonk** [comment]
I agree with showing edits and making better message deletion more customisable - however I likely wouldn't add a full proper message logger, sorry. Will change issue scope to improving better message deletion, and if you want you can make a new one for edits (which would likely be another separate module).
